authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import ProfileRetrieveAPIView
+
+app_name = 'profiles'
+urlpatterns = [
+    path('profiles/<str:username>', ProfileRetrieveAPIView.as_view()),
```

Comment:
Please add a `/` at the end of this URL and make it protected as well. Follow the implementation that was used in creating articles.
If you're able to before 10 a. m tomorrow, add a protected URL to retrieve all profiles as well.

Reply:
okay on it

Reply:
I have added this URL please check it out